app/models/move.rb

```diff
@@ -46,27 +45,21 @@ class Move < VersionedModel
   has_many :events, as: :eventable, dependent: :destroy # NB: polymorphic association
 
   validates :from_location, presence: true
-  validates(
-    :to_location,
-    presence: true,
-    unless: ->(move) { move.move_type == 'prison_recall' },
-  )
+  validates :to_location, presence: true, unless: :prison_recall?
   validates :move_type, inclusion: { in: move_types }
-  validates :person, presence: true, unless: -> { [MOVE_STATUS_REQUESTED, MOVE_STATUS_CANCELLED].include?(status) }
+  validates :person, presence: true, unless: -> { requested? || cancelled? }
   validates :reference, presence: true
 
   # we need to avoid creating/updating a move with the same person/date/from/to if there is already one in the same state
   # except that we need to allow multiple cancelled moves
   validates :date, uniqueness: { scope: %i[status person_id from_location_id to_location_id] },
-            unless: -> { [MOVE_STATUS_PROPOSED, MOVE_STATUS_CANCELLED].include?(status) || person_id.blank? }
-  validates :date, presence: true,
-            unless: -> { status == MOVE_STATUS_PROPOSED }
-  validates :date_from, presence: true,
-            if: -> { status == MOVE_STATUS_PROPOSED }
-
-  validate :date_to_after_date_from
-
+            unless: -> { proposed? || cancelled? || person_id.blank? }
+  validates :date, presence: true, unless: -> { proposed? || cancelled? }
```

Comment:
could you add a test for the `cancelled` validation here?

Reply:
will do 👍